application-analytics-default/src/main/java/com/xwiki/analytics/internal/RowEvolutionJsonNormaliser.java

```diff
@@ -60,11 +60,12 @@ public String getIdentifier()
     }
 
     /**
-     * Transforming the json object returned by Matomo into an array of jsons to make it easier to use in javascript.
+     * Transforming the json object returned by Matomo into an array of jsons to make it easier to use in javascript and
+     * adding a date to each entry to be able to idenitfy for what period the statistics are.
      *
-     * @param jsonNode
+     * @param jsonNode the response from Matomo that is a json
      * @param filters holds the criteria for filtering a dataset
-     * @return filtered array of jsons
+     * @return filtered array of processed jsons with the date added to them
      */
     protected ArrayNode processObjectNode(JsonNode jsonNode, Map<String, String> filters) throws JsonProcessingException
```

Comment:
Missing `@Override`
I know it is not mandatory, but it is very helpful to not do a mistake in the method signature

Also, when you do the override, the comments are not mandatory to be added again. You should do this if you think there is something more to say than on the original method